Lib/idlelib/pyshell.py

```diff
@@ -1698,6 +1698,8 @@ def main():
         root.mainloop()
```

Comment:
The clipboard content can be lost if `mainloop()` raise an exception. Use `try`/`finally` to always call `_finalize_tcl()`.

Reply:
Good call.

Reply:
Done.